core/.storybook/__mocks__/Stages.actions.mock.ts

```diff
@@ -0,0 +1,3 @@
+import { fn } from "@storybook/test";
+
+export const move = fn();
```

Comment:
I couldn't get this to work, even though as far as I could tell I mimicked the same mocks as `PubEditor.actions.mock.ts` 🥲